themes/classic/modules/ps_shoppingcart/modal.tpl

```diff
@@ -56,12 +56,20 @@
               {else}
                 <p class="cart-products-count">{l s='There is %product_count% item in your cart.' sprintf=['%product_count%' =>$cart.products_count] d='Shop.Theme.Checkout'}</p>
               {/if}
-              <p><strong>{l s='Total products:' d='Shop.Theme.Checkout'}</strong>&nbsp;{$cart.subtotals.products.value}</p>
-              <p><strong>{l s='Total shipping:' d='Shop.Theme.Checkout'}</strong>&nbsp;{$cart.subtotals.shipping.value} {hook h='displayCheckoutSubtotalDetails' subtotal=$cart.subtotals.shipping}</p>
+              <p><span><strong>{l s='Total products:' d='Shop.Theme.Checkout'}</strong></span>&nbsp;<span><strong>{$cart.subtotals.products.value}</strong></span></p>
```

Comment:
Why so many `span` and `strong`?
How about using only `span` and adding a named class? It will also help for creating automated tests.
```suggestion
              <p class="total-products-line"><span class="label">{l s='Total products:' d='Shop.Theme.Checkout'}</span>&nbsp;<span class="value">{$cart.subtotals.products.value}</span></p>
```

This could be done for the other lines as well

Reply:
Also is this the total number of products or the total amount for products? I think that if it's an amount it should be `Products total`

/cc @LouiseBonnard

Reply:
<p class="total-products-line"><span class="label">{l s='Subtotal' d='Shop.Theme.Checkout'}</span>&nbsp;<span class="value">{$cart.subtotals.products.value}</span></p>

Reply:
I changed previous wording with "Subtotal"